.env.production

```diff
@@ -3,7 +3,7 @@ RECORD_MANAGER_APP_TITLE=Record Manager
 RECORD_MANAGER_APP_INFO=
 RECORD_MANAGER_LANGUAGE=cs
 RECORD_MANAGER_NAVIGATOR_LANGUAGE=true
-RECORD_MANAGER_BASENAME=/
+RECORD_MANAGER_BASENAME=/record-manager
```

Comment:
What is motivation to change this?

Reply:
Because I thought of every environment (dev, production) having it's own _base_ in vite.config.js which will use the value of the RECORD_MANAGER_BASENAME based on mode. So the dev environment will have a **"/" base**, and the production will have a _"/record-manager" base_ . Since the build for production image is then used in dockerization, it means that the **base of the build** (in vite.config.js) should match with the BASENAME passed via docker (which will be /record-manager, since RECORD_MANAGER_ROOT_PATH in deploy > internal-auth > .env is commented out). [https://reactrouter.com/en/main/router-components/router](url)

Reply:
So what if i decide to have RECORD_MANAGER_ROOT_PATH=my-record-manager ?

Reply:
@blcham  It doesn't work when I change it, I feel stuck now